src/pages/contact-us/_contact-ways.js

```diff
@@ -0,0 +1,140 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Text } from '../../components/elements/typography'
+import Show from 'components/containers/show'
+import { Header } from 'components/elements'
+import { localize, LocalizedLink } from 'components/localization'
+import CallUsIcon from 'images/svg/call-us.svg'
+// import ChatLiveIcon from 'images/svg/chat-live.svg'
+import EmailUsIcon from 'images/svg/email-us.svg'
+import device from 'themes/device'
+
+const Wrapper = styled.section`
+    width: 100%;
+    height: 44.5rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    flex-wrap: nowrap;
+    background-color: var(--color-white);
+    padding: 4rem 12.8rem;
+
+    @media ${device.tabletL} {
+        flex-wrap: wrap;
+        height: auto;
+    }
+`
+const Contact = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    flex-wrap: nowrap;
+    width: 38.4rem;
+    padding-top: 3.2rem;
+`
+
+const StyledHeader = styled(Header)`
+    margin-bottom: 2.4rem;
+`
+const StyledText = styled(Text)`
+    margin-bottom: ${props => props.marginBttom || '0.8rem'};
+    color: ${props => props.color || 'var(--color-black-3)'};
+`
+const Logo = styled.div`
+    margin-bottom: 1.4rem;
+`
+const StyledLink = styled(LocalizedLink)`
+    text-decoration: none;
+    transform: rotate(45deg);
+    border: solid var(--color-red);
+    border-width: 0 2px 2px 0;
+    display: inline-block;
+    padding: 4px;
+    cursor: pointer;
+`
+const Splitter = styled.div`
+    background-color: #d8d8d8;
+    height: 24.3rem;
+    width: 0.1rem;
+    margin-top: 7.8rem;
+`
+// const StyledButton = styled(LocalizedLink)`
```

Comment:
Why is this commented out?

Reply:
Because it will be added shortly to this section, but for now we cannot have it due to back-end requirements.

Reply:
Ok 👍 could you add a short TODO comment also?